src/Agent.php

```diff
@@ -102,4 +104,24 @@ public function getRequestStartTime(): float
     {
         return $this->request_start_time;
     }
+
+    public function send(): bool
+    {
+        $sent = parent::send();
+
+        // Ensure collectors are reset after data is sent to APM
+        $this->collectors->each(function (EventDataCollector $collector) {
+            $collector->reset();
+        });
+
+        /*
+         * Push new metadata onto the stack in preparation for the next send event. This
+         * simulates the behavior when a new Agent is created and is needed for long running
+         * worker processes. A future release of the Agent package should handle event
+         * collection better and remove the need for this.
+         */
+        $this->putEvent(new Metadata([], $this->getConfig(), $this->agentMetadata()));
```

Comment:
```suggestion
        $this->putEvent(new Metadata([], $this->getConfig()));
```

Reply:
Fixed. I actually installed the old package this time. Should be good. Sorry about that.